code_style.sh

```diff
@@ -40,10 +40,11 @@ find_cfiles() {
        ! \( $(get_find_path_args $(trim_comments .clang-format-ignore)) \) "$@"
 }
 
-# We're targeting clang-format version 17 and other versions give slightly
-# different results, so prefer `clang-format-17` if it's installed.
+# We're targeting clang-format version 20 and other versions give slightly
+# different results, so prefer `clang-format-20` if it's installed.
+llvm_version=20
 clang_format_exe_names=(
-  'clang-format-17'
+  "clang-format-${llvm_version}"
```

Comment:
We don't need `llvm_version` at the moment.
Let's keep it as `clang-format-20` for easier find & replace.

Reply:
no:
- it's harder to find
- it's not easier, it's 1 line versus 1 line

Reply:
Eh, if you insist.